katharsis-core/src/main/java/io/katharsis/utils/ClassUtils.java

```diff
@@ -16,9 +16,6 @@
  */
 public class ClassUtils {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)